portalnet/src/storage.rs

```diff
@@ -399,18 +395,12 @@ impl PortalStorage {
             });
         }
 
-        // Store the data in radius db
-        self.db_insert(&content_id, value)?;
+        // Store the data in db
         let content_key: Vec<u8> = key.clone().into();
         // store content key w/o the 0x prefix
         let content_key = hex_encode(content_key).trim_start_matches("0x").to_string();
-        // Revert rocks db action if there's an error with writing to metadata db
-        if let Err(err) = self.meta_db_insert(&content_id, &content_key, value) {
-            debug!(
-                "Error writing content ID {:?} to meta db. Reverting: {:?}",
-                content_id, err
-            );
-            self.db.delete(content_id)?;
+        if let Err(err) = self.db_insert(&content_id, &content_key, value) {
+            debug!("Error writing content ID {:?} to db: {:?}", content_id, err);
```

Comment:
These args can be inlined, so I'm curious why our [lint](https://github.com/ethereum/trin/blob/master/portalnet/src/lib.rs#L2) isn't picking up on it?